docs/pages/en-us/Architecture.md

```diff
@@ -18,24 +18,28 @@ The Data Processing service is actually a suite of services that form a fully-fe
 - Workers, built against the CAF Worker Framework, for processing input data.
 - API's for customizing logic that can be applied during processing, for example, workflows and boilerplate expressions.
 
-The sections that follow provide a high-level architectural overview of the various components that make up the Data Processing service and how they interact. Each component is available as a Docker container and the suite can be deployed using the compose file available [here](https://github.com/CAFDataProcessing/data-processing-service-deploy). Instructions on using the compose file can be found on the [Getting Started](./Getting-Started) page.
+The sections that follow provide a high-level architectural overview of the various components that make up the Data Processing service and how they interact. Each component is available as a Docker container and the suite can be deployed using the compose file available [here](https://github.houston.softwaregrp.net/caf/data-processing-service-internal-deploy/tree/deploy). Instructions on using the compose file can be found on the [Getting Started](./Getting-Started) page.
 
 ## Overview
 
 The Data Processing service consists of web service API's, workers built against the Worker Framework and databases to store information. Workers and API's are state-less in this system and thus can scale using the auto-scaler to accommodate the workload at any point in time. Tasks are sent to workers through messages on RabbitMQ queues and the results come as messages to designated queues by each worker.
 
-The figure below illustrates the overall flow and relationship of components in Data Processing.
+The figure below illustrates the overall flow and relationship of components in Data Processing. For the sake of reducing the size of the diagram only some of the processing workers are represented but all the processing workers should behave in the same fashion. 
 
-![Overall System](../../assets/img/Architecture/overall.png)
+![Overall System](../../assets/img/Architecture/chaining_overall.png)
```

Comment:
Check in the source svg file as well as the rendered png file.

Reply:
Done.